samples/README.md

```diff
@@ -0,0 +1,83 @@
+# Data Directory Organization
+
+Each TIDES Data Package example should follow the following directory structure, consistent with the structure of the [Frictionless Data Package specification](https://specs.frictionlessdata.io/data-package/), including:
+
+```
+unique-example-name
+  \TIDES     # data to be validated against the TIDES specification
+  \TIDES\datapackages.json # data package metadata per https://specs.frictionlessdata.io/data-package/
+  \raw      # data which the agency uses to create TIDES data
+  \scripts  # scripts used to transform raw --> TIDES
+```
+
+## Adding Examples
+
+We encourage the addition of examples, but please follow the following guidelines:
+
+1. *No large files* This isn't the place to store your data, rather to document some minimal examples.  The recommended size is 100-1000 records per file, more if absolutely required to reproduce an issue with the spec.  All individual files should be well under 50 MB.  
+2. *Include Metadata* as specified in [`datapackage.json`](#data-package).
+3. *Include a README.md* in the base folder of your example with an overview so that it can be included in the documentation.
+
+## Data Package
+
+TIDES data packages must include a [`datapackage.json`](https://specs.frictionlessdata.io/data-package/).  Key information to include in [`datapackage.json`](https://specs.frictionlessdata.io/data-package/) includes:
+
+| **Field** | **Description** | **Required** |
+| --------- | --------------- | ------------ |
+| `title` | A human-readable title. | Required |
+| `name` | Identifier string as a URL-friendly slug. | Required |
+| `description` | Short description of data package. | Recommended |
+| `agency` | Transit agency name. | Recommended |
+| `ntd_id` | ID for the National Transit Database. | Recommended |
+| `profile` | Should be `tabular-data-package` | Required |
+| `licenses` | Should be `[{"name": "Apache-2.0"}]` to be consistent with this repository | Required |
+| `contributors` | Array of data contributors `[{"title": "My Name", "github": "my_handle", "email": "me@myself.com"}]` | Recommended |
+| `maintainers` |  Array of data maintainers `[{"title": "My Name", "github": "my_handle", "email": "me@myself.com"}]` | Recommended |
+| `resources` | Array of data files included in your package, formated as a [`tabular-data-resource`](#data-resource)| Required |
+
+### Template
+
+A `datapackage.json` template is available at [`/data/template/TIDES/datapackage.json`](https://raw.githubusercontent.com/TIDES-transit/TIDES/main/data/template/TIDES/datapackage.json).
+
```

Comment:
I don't see why this is better when we can include the file in the markdown? I'd prefer to manage data as data, not as data within markdown.

Reply:
But it's not data!—it's documentation, an example that doesn't even have meaningful values. Documentation for just about every software tool is [littered](https://docs.opentripplanner.org/en/latest/BuildConfiguration/) with [example](https://en.wikibooks.org/wiki/OpenSSH/Client_Configuration_Files#System-wide_Client_Configuration_Files) [code](https://docs.python.org/3/library/os.html#os.scandir) and [configuration](https://git-scm.com/docs/git-config#EXAMPLES), this is exactly the same.

If you want to include a file, go ahead, but I do think that at the very least the file should have an obviously different name, e.g., `datapackage.json.template` or `datapackage.sample` or something like that so that it's not accidentally interpreted as a real data package description.